### ISSUE TITLE
W3C "Spatial Data on the Web Use Cases & Requirements" NOTE

### ISSUE THREAD

**Issue by dret** [geojson/draft-geojson#88]
this one may be interesting to look at, and maybe it's even useful to add this as a reference? http://www.w3.org/TR/2015/NOTE-sdw-ucr-20150723/

**Comment by adoyle** [comment]
I’d be -1 on this. Overall, that document seems to be taking a shotgun approach with many people throwing in their favorite bits. I’d hate to imagine the outcome if a committee were to design something based on it. Unless, of course, they just decide that GML is the answer. In which case it’s no longer relevant to GeoJSON.

Also, GeoJSON satisfies some of the requirements, so they are more likely to reference GeoJSON and use-cases are meant to inform designs. GeoJSON is way past design.

The reference would have to be non-normative. I’d rather not send readers of the GeoJSON spec off into other documents that are more likely to be confusing than not.

	Allan

**Comment by dret** [comment]
> On Jul 23, 2015, at 10:05, Allan Doyle <notifications@github.com> wrote:
> I’d be -1 on this. Overall, that document seems to be taking a shotgun approach with many people throwing in their favorite bits. I’d hate to imagine the outcome if a committee were to design something based on it. Unless, of course, they just decide that GML is the answer. In which case it’s no longer relevant to GeoJSON.

fair enough. and just for the record: this was not intended as an endorsement of the note. i just wanted to point it out as some new and potentially relevant work.

**Comment by adoyle** [comment]
> On Jul 23, 2015, at 4:05 PM, Erik Wilde <notifications@github.com> wrote:
> 
> > On Jul 23, 2015, at 10:05, Allan Doyle <notifications@github.com> wrote:
> > I’d be -1 on this. Overall, that document seems to be taking a shotgun approach with many people throwing in their favorite bits. I’d hate to imagine the outcome if a committee were to design something based on it. Unless, of course, they just decide that GML is the answer. In which case it’s no longer relevant to GeoJSON.
> 
> fair enough. and just for the record: this was not intended as an endorsement of the note. i just wanted to point it out as some new and potentially relevant work.

No, I didn’t take it as an endorsement. And it is potentially relevant. Maybe it can be referenced by the WG in some way. I just don’t see needing to reference if from the spec.  

I’m sure once the W3C and OGC get outside comments they will tighten up the document.

**Comment by sgillies** [comment]
Engagement with the W3C and OGC will be part of our charter. I think we can satisfy that by announcements in that WG's forums and intercepting work that duplicates what we're doing. For example, reminding them to consider and contribute to the temporal framework in https://github.com/geojson/geojson-ld instead of rolling a new one.

**Comment by dret** [comment]
i'd be cautious with the "intercepting" aspect, which easily can turn into something that's more a question of preferences and strategy than one of focused technical work.
and i'd also recommend to separate GeoJSON and the LD variant very well, so that progress on the former is not held up by any issues coming up from the LD side of things. LD should be a clearly and cleanly layered spec: of course GeoJSON should be a solid and well-defined foundation (in particular regarding openness and extensibility, see #56 #57 #59 #60), but GeoJSON itself should not have any dependency on GeoJSON-LD.

**Comment by dret** [comment]
after posting the pointer yesterday i took a little bit of time to look through the document. wow. it indeed looks like a rather random grab-bag of anything vaguely location-related. it'll be interesting to see what comes out of this very broad approach.

**Comment by dr-shorthair** [comment]
I mentioned these observations in SDW telecon this evening. The document you are referring to is a 'Request for Comments' in the true sense, so if this group has any constructive suggestions they would be welcomed. Please mailto:public-sdw-comments@w3.org
